app/src/main/java/com/ruthvikbr/starbucksindiacompose/ui/screens/home/HomeViewModel.kt

```diff
@@ -41,4 +48,10 @@ class HomeViewModel @Inject constructor(
             _popularMenuItemsList.value = fetchPopularMenuItemsUseCase()
         }
     }
+
+    private fun fetchStarbucksNewsItems() {
+        viewModelScope.launch {
```

Comment:
@Ruthvikbr how are we doing error handling here ?

Reply:
@oianmol I have made use of CoroutineExceptionHandler as suggested